pii.go

```diff
@@ -0,0 +1,15 @@
+package stripe
+
+import (
+  "net/url"
+)
+
+type PIIParams struct {
+  Params
+  PersonalIdNumber string
```

Comment:
If you look elsewhere in the bindings, it seems that convention is to upcase "ID" (e.g. `ID` under the `Charge` type). Can you change that?